docs/development_suite/identity-and-access-management/group-management.md

```diff
@@ -0,0 +1,138 @@
+---
+id: group-management
+title: Manage Groups
+sidebar_label: Manage Groups
+---
+
+Mia-Platform Console allows to assign and manage authorization privileges to groups of users at Company level. Groups aim at simplify the management a large number of users who have to be assigned the same privileges (e.g. a work team).
+
+<!-- 
+
+ITA version: 
+Mia-Platform Console permette di assegnare e gestire privilegi di autorizzazione a gruppi di utenti presenti o meno a livello di Company. In questo modo risulta più agevole gestire un grande numero di utenti a cui dover assegnare gli stessi privilegi (es. work team).
+
+
+Mia-Platform Console enables certain Users to control other identities ability to access resources. This grants them ownership over the decision-making process regarding which resources are accessible and at what level of authorization. -->
+
+
+Groups is considered a type of [Identity](/development_suite/identity-and-access-management/overview.md#identity-and-access-management) to which Users with enough permissions can assign and manage roles on the following resources:
+
+* Company
+* Project
+* Runtime Environment
+
+:::note
+To find out more about Roles management, check out the available [Capabilities](/development_suite/identity-and-access-management/console-levels-and-permission-management.md#users-capabilities-inside-console) that can be assigned to an Identity.
+:::
+
+
+<!-- In particular, a User can be assigned a Role on the following resources:
+
+* Company
+* Project
+* Runtime Environment -->
+
+
+<!-- Only Users with enough privileges will be granted access to the Identities administration portal and will be able to change identity Roles. -->
+
+## Managing Company Groups
+<!-- A User with enough administrative permission on a specific Company will be able to view the existing identities in the Company, add new ones and change the existing identities' Roles. -->
+
+Each Company has its own Groups, which are configurable and manageable solely by Company Owner from the dedicated Groups section inside the Company Overview. 
+
+At Company level, for each Group, it is possible to manage:
+
+* Name: the name assigned to the Group
+* Role: the Group Company Role
+* Members: adding and/or removing Users from the Group
+
+Existing Groups can also be found within the Identities portal at both Company and Project level, where it is possible to manage Roles assigned to them as well as for Users and Service Account Users. 
+
+<!-- TODO: ![Group table](./img/group-management/group_table.png) -->
+
+:::caution
+Please note that some permissions defined by the Company Role may be inherited on the Projects and Runtime Environment owned by the Company itself.  
+Always pay attention when assigning Roles in order to avoid providing undesired access to resources!
```

Comment:
it may be helpful to put also here a link to the Console levels and permission management page